### ISSUE TITLE
Add metabox to specify active menu item

### ISSUE THREAD

**Issue by natejacobson** [washingtonstateuniversity/WSUWP-spine-parent-theme#85]
There will be many cases in which an item is buried too deep or for other reasons, shouldn't be in navigation. However, when visited, we will want a parent item indicated within the nav. Metabox would have a drop down of existing nav and enable the selection of one and only one active item.

**Comment by jeremyfelt** [comment]
We should take a look at this in the nearer future.

**Comment by natejacobson** [comment]
Yes. Unfortunately, this will be especially tricky when the nav is being fed from a network location and not the site itself. And then there's integration with BU Navigation. The best solution may be JS.

**Comment by natejacobson** [comment]
Actually, if instead of nice user friendly menu item selector we just provided the ability to enter a menu item ID, that might work.

**Comment by jeremyfelt** [comment]
When there's a will, there's a way. And we control the filters for it all. ;)
